Alliance.Client/Core/Handlers/ClientCommonHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Alliance.Client.Extensions.RTSCamera.Views;
+using Alliance.Common.Extensions;
+using Alliance.Common.Extensions.AdminMenu.NetworkMessages.FromServer;
+using TaleWorlds.MountAndBlade;
+
+namespace Alliance.Server.Core.Handlers
+{
+	public class ClientCommonHandler : IHandlerRegister
+	{
+		public void Register(GameNetwork.NetworkMessageHandlerRegisterer reg)
+		{
+			reg.Register<SetClientCameraFrame>(HandleClientCameraFrame);
+		}
+
+		private void HandleClientCameraFrame(SetClientCameraFrame setClientCameraFrameMsg)
+		{
+			Mission.Current.GetMissionBehavior<RTSCameraView>().ForceCameraToFrame(setClientCameraFrameMsg);
```

Comment:
Je rajouterai juste un null check au cas où la mission est pas en cours ou que RTSCameraView ne soit pas chargé (en changement de mission par exemple, si on reçoit un message avec du retard).

```suggestion
			Mission.Current?.GetMissionBehavior<RTSCameraView>()?.ForceCameraToFrame(setClientCameraFrameMsg);
```

Reply:
Bien vu, c'est ajouté !